app/forms/filters/pension_pot.rb

```diff
@@ -5,9 +5,11 @@ module PensionPot
     attr_accessor :pension_pot_size
 
     def options_for_pension_pot_sizes
-      InvestmentSize.all.map do |investment_size|
+      [
+        [I18n.t('search_filter.pension_pot.any_size_option'), ANY_SIZE_VALUE]
+      ] + InvestmentSize.all.map do |investment_size|
         [investment_size.localized_name, investment_size.id]
-      end << [I18n.t('search_filter.pension_pot.any_size_option'), ANY_SIZE_VALUE]
```

Comment:
Not a blocker, just wondering whether using `.unshift` method would be a tidier way of adding the size option to the start of the array? 

`end.unshift([I18n.t('search_filter.pension_pot.any_size_option'), ANY_SIZE_VALUE])`